src/OrleansTestingHost/TestingSiloHost.cs

```diff
@@ -657,9 +657,8 @@ private SiloHandle StartOrleansSilo(Silo.SiloType type, TestingSiloOptions optio
         /// <param name="type">The type of the silo to deploy</param>
         /// <param name="options">The options to use for the silo</param>
         /// <param name="instanceCount">The instance count of the silo</param>
-        /// <param name="shared">The shared AppDomain to use</param>
         /// <returns>A handle to the deployed silo</returns>
-        public static SiloHandle StartOrleansSilo(TestingSiloHost host, Silo.SiloType type, TestingSiloOptions options, int instanceCount, AppDomain shared = null)
+        public static SiloHandle StartOrleansSilo(TestingSiloHost host, Silo.SiloType type, TestingSiloOptions options, int instanceCount)
```

Comment:
Are any of these needed? Otherwise please undo as they seem unrelated to the PR. If there are tests that use TestingSiloHost, we need to update them to TestCluster before porting them to vNext. We already agreed that we are not porting TestingSiloHost to vNext, remember?
I think this unnecessary breaking change could affect some of our VSO tests that use the TestingSiloHost directly. But still, since we are going to deprecate TestingSiloHost, let's avoid it.